pkg/domain/entities/images.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/containers/podman/v4/pkg/inspect"
 	"github.com/containers/podman/v4/pkg/trust"
 	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/pkg/jsonmessage"
```

Comment:
Let's copy the type over here.  This way we have full control and avoid potential silent regressions.

Reply:
Only the type(s) and methods that are needed.

Reply:
It is also used in compat API. Are you sure you want to move and leave there untouched?

Reply:
Yes, thanks for double checking.  The reasoning for the compat API is to remain compatible and follow the types.

For libpod, we should control the types.